src/nvim/testdir/runnvim.sh

```diff
@@ -38,22 +40,26 @@ main() {(
   then
     fail "$test_name" F "Nvim exited with non-zero code"
   fi
-  echo "Stdout of :terminal runner" >> "$tlog"
-  echo "$separator" >> "$tlog"
-  cat "out-$tlog" >> "$tlog"
-  echo "$separator" >> "$tlog"
-  echo "Stderr of :terminal runner" >> "$tlog"
-  echo "$separator" >> "$tlog"
-  cat "err-$tlog" >> "$tlog"
-  echo "$separator" >> "$tlog"
+  {
+    echo "Stdout of :terminal runner"
+    echo "$separator"
+    cat "out-$tlog"
+    echo "$separator"
+    echo "Stderr of :terminal runner"
+    echo "$separator"
+    cat "err-$tlog"
+    echo "$separator"
+  } >> "$tlog"
```

Comment:
IIRC it's good practice to put it at the start.

```
>> "$tlog" {
  ...
}
```

Reply:
Hmm, that does not work.. "syntax error"